MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/PodInstall/Info.plist

```diff
@@ -41,5 +41,7 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSLocationAlwaysUsageDescription</key>
```

Comment:
This key is ignored in iOS 11.0 and above. You need `NSLocationAlwaysAndWhenInUseUsageDescription` instead:

https://developer.apple.com/documentation/corelocation/choosing_the_authorization_level_for_location_services/requesting_always_authorization